### PR TITLE
Bugfix - Live Chart Nodes Search Tags

### PR DESCRIPTION
**Pull request by dnenov**
### Purpose

A small PR modifying the Search Tags of a series of Nodes. 

The recently added Live Chart nodes had duplicate information in Search Tags and the AlsoKnownAs node properties, which was causing an error to be thrown when adding the node information to the Search Dictionary. Changing the Search Tags solved this issue.

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [x] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

- `search tags` were duplicating the `alsoknownas` property causing an error to be thrown

### Reviewers

@mjkkirschner 
@reddyashish 

### FYIs

@Amoursol

### CODE REVIEW

File: src/Libraries/CoreNodeModelsWpf/Properties/CoreNodeModelWpfResources.en-US.resx [hunk 1]
```diff
@@ -121,37 +121,57 @@
...
+List&lt;string&gt;</value>
+  </data>
+  <data name="ChartsPieChartValuesDataPortToolTip" xml:space="preserve">
+    <value>A list of double values to supply a value for each segment of the pie chart.
+
+List&lt;double&gt;</value>
+  </data>
+  <data name="ChartsPieChartColorsDataPortToolTip" xml:space="preserve">
+    <value>A list of colors for each segment of the pie chart.
+
```
Comment (reddyashish) [inline src/Libraries/CoreNodeModelsWpf/Properties/CoreNodeModelWpfResources.en-US.resx:156]: Add these new resources to the other resx file.

**Comment by QilongTang** [comment]
@reddyashish Side question, are search tags put into resources so that they can be surfaced in a localized env?

**Comment by dnenov** [comment]
> @reddyashish Side question, are search tags put into resources so that they can be surfaced in a localized env?

The search tags are in resources @QilongTang , yes!

**Comment by mjkkirschner** [comment]
new test failure:

DynamoCoreWpfTests.DocumentationBrowserViewExtensionContentTests.CheckThatEmbeddedHtmlContentDoesNotSanitize("D:\\jenkins\\workspace\\Dynamo\\DynamoSelfServe\\pullRequestValidation\\Dynamo\\src\\DocumentationBrowserViewExtension\\Docs\\NoContent.html")
--


<br class="Apple-interchange-newline">[DynamoCoreWpfTests.DocumentationBrowserViewExtensionContentTests.CheckThatEmbeddedHtmlContentDoesNotSanitize("D:\\jenkins\\workspace\\Dynamo\\DynamoSelfServe\\pullRequestValidation\\Dynamo\\src\\DocumentationBrowserViewExtension\\Docs\\NoContent.html")](https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/pullRequestValidation/8727/testReport/junit/DynamoCoreWpfTests/DocumentationBrowserViewExtensionContentTests/CheckThatEmbeddedHtmlContentDoesNotSanitize__D___jenkins__workspace__Dynamo__DynamoSelfServe__pullRequestValidation__Dynamo__src__DocumentationBrowserViewExtension__Docs__NoContent_html__/)

**Comment by dnenov** [comment]
> new test failure:
> 
> https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/pullRequestValidation/8727/testReport/junit/DynamoCoreWpfTests/DocumentationBrowserViewExtensionContentTests/CheckThatEmbeddedHtmlContentDoesNotSanitize__D___jenkins__workspace__Dynamo__DynamoSelfServe__pullRequestValidation__Dynamo__src__DocumentationBrowserViewExtension__Docs__NoContent_html__/

This test is passing locally. I tried rebuilding the PR, but it popped up with 2 more failing tests. I might get a new PR from scratch based on the latest Dynamo build.

**Comment by mjkkirschner** [comment]
@dnenov - you can ignore https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/pullRequestValidation/8727/testReport/junit/DynamoCoreWpfTests/DocumentationBrowserViewExtensionContentTests/CheckThatEmbeddedHtmlContentDoesNotSanitize__D___jenkins__workspace__Dynamo__DynamoSelfServe__pullRequestValidation__Dynamo__src__DocumentationBrowserViewExtension__Docs__NoContent_html__/ - it's from the previous commit to master.

**Review by reddyashish — APPROVED** [review]
Will merge once the self-serve finishes

**Pull request merged**